src/emqx_cm.erl

```diff
@@ -149,8 +153,11 @@ connection_closed(ClientId) ->
     connection_closed(ClientId, self()).
 
 -spec(connection_closed(emqx_types:clientid(), chan_pid()) -> true).
-connection_closed(ClientId, ChanPid) ->
-    ets:delete_object(?CHAN_CONN_TAB, {ClientId, ChanPid}).
+connection_closed(_ClientId, _ChanPid) ->
```

Comment:
delete_object never worked anyway, right? 
Should we keep this function at all?

Reply:
I kept it so that the contract with `emqx_cm` does not change.